volume/local/local.go

```diff
@@ -268,7 +268,7 @@ func (r *Root) validateName(name string) error {
 		return validationError{fmt.Errorf("volume name is too short, names should be at least two alphanumeric characters")}
 	}
 	if !volumeNameRegex.MatchString(name) {
-		return validationError{fmt.Errorf("%q includes invalid characters for a local volume name, only %q are allowed", name, utils.RestrictedNameChars)}
+		return validationError{fmt.Errorf("%q includes invalid characters for a local volume name, only %q are allowed. If you intented to pass host directory use absolute path starting with /", name, utils.RestrictedNameChars)}
```

Comment:
@muayyad-alsadi First of all, thank you so much for creating this Pull request ☺️👌

@cpuguy83 Idk, [Wikipeida says](https://en.wikipedia.org/wiki/Path_(computing)#Absolute_and_relative_paths): _«An absolute or full path points to the same location in a file system, regardless of the current working directory. To do that, it must include the root directory.»_, so I guess only absolute would work, too, since `./foo` is dependent on the working directory.

However, you can also use absolute paths starting with `~/` (home directory), so we should add this to the message, too?

Maybe we could settle for "If you intended to pass a host path, use an absolute path  starting with / or ~"?